README.md

```diff
@@ -48,6 +48,7 @@ Contributions are always welcome. Please read the [Contribution Guidelines](CONT
 - [Jailbreak Chat](https://www.jailbreakchat.com/)
 - [Adversarial Prompting](https://www.promptingguide.ai/risks/adversarial)
 - [Epivolis](https://epivolis.com/): a prompt injection aware chatbot designed to mitigate adversarial efforts
+- [LLM Security Problems at DEFCON](https://github.com/Nautilus-Institute/quals-2023/tree/main/pawan_gupta): the world's top security competition
```

Comment:
Can you add the version(?) of DEFCON like this?

 ```suggestion
- [LLM Security Problems at DEFCON31 Quals](https://github.com/Nautilus-Institute/quals-2023/tree/main/pawan_gupta): the world's top security competition
```

Reply:
thanks for your good idea!